pkgs/servers/oauth2_proxy/default.nix

```diff
@@ -2,23 +2,28 @@
 
 buildGoPackage rec {
   pname = "oauth2_proxy";
-  version = "3.2.0";
-  
+  version = "5.1.0";
+
   goPackagePath = "github.com/pusher/${pname}";
 
   src = fetchFromGitHub {
     repo = pname;
     owner = "pusher";
```

Comment:
This has been redirected to oauth2-proxy/oauth2-proxy.

Reply:
Yes, it has been, but not official yet. I'd wait for them to make a release and then do the change. That is, Go code still uses pusher/oauth2-proxy, while only the repo is renamed.